### PR TITLE
Display Lineups in GUI.

### PR DESCRIPTION
**Pull request by FYimu**
Update Person settings to include lineup names as attributes.
Display lineups a person belongs to in the same way as tags.
This pull request solves #58

### CODE REVIEW

**Review by snoidetx — APPROVED** [review]
These codes successfully differentiate tags and lineups by using a different colour. Well done!

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -137,6 +166,7 @@ public int hashCode() {
     @Override
```
Comment (brandonrhan) [inline src/main/java/seedu/address/model/person/Person.java:166]: We might need a javadoc comment here

File: src/main/java/seedu/address/storage/JsonAdaptedLineupName.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     * Converts a given {@code LineupName} into this class for Jackson use.
+     */
+    public JsonAdaptedLineupName(LineupName lineupName) {
+        this.lineupName = lineupName.lineupName;
+    }
+
+    @JsonValue
+    public String getLineupName() {return lineupName;}
+
+    public LineupName toModelType() throws IllegalValueException {
```
Comment (brandonrhan) [inline src/main/java/seedu/address/storage/JsonAdaptedLineupName.java:35]: Same for here

**Review by brandonrhan — COMMENTED** [review]
Well done just some JavaDoc comments are missing

**Pull request merged**